template/python3-http-debian/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM openfaas/of-watchdog:0.7.2 as watchdog
```

Comment:
```suggestion
FROM openfaas/of-watchdog:0.7.6 as watchdog
```

Reply:
We have 0.7.7 now but this can be changed after. https://github.com/openfaas-incubator/of-watchdog/releases/tag/0.7.7